superset/assets/src/explore/components/controls/SubscriberLayer.jsx

```diff
@@ -341,10 +351,21 @@ export default class SubscriberLayer extends React.PureComponent {
                 onChange={v => this.setState({ name: v })}
                 validationErrors={!name ? [t('Mandatory')] : []}
               />
+
+              <CheckboxControl
+              hovered
+                name="annotation-layer-hide-line"
+                label="Use as popup"
```

Comment:
name should be  different annotation-layer-hide-line it may used anywhere in component

Reply:
updated name